src/sections/contact-section.html

```diff
@@ -0,0 +1,38 @@
+<div class="pt-[6%] container grid text-2xl grid-rows-2 md:grid-cols-2 gap-16 md:gap-20">
```

Comment:
Is there a specific reason to use arbitrary value here?

Reply:
Tailwind doesn't offer utility classes for small percentages like 6%, so we decided to use an arbitrary value `pt-[6%]` for it to respond on different screens.